test/dataframerow.jl

```diff
@@ -42,27 +47,41 @@ end
 
     r = DataFrameRow(df, 2, :)
     @test r[:] === r
+    @test isequal(r[r""], r)
```

Comment:
We use `≈` consistently I think. Same below.

Reply:
fixed